crates/rooch-indexer/src/proxy/mod.rs

```diff
@@ -36,6 +36,27 @@ impl IndexerProxy {
         }
     }
 
+    pub async fn update_indexer(
+        &self,
+        root: RootObjectEntity,
+        ledger_transaction: LedgerTransaction,
+        execution_info: TransactionExecutionInfo,
+        moveos_tx: VerifiedMoveOSTransaction,
+        events: Vec<Event>,
+        state_change_set: StateChangeSet,
+    ) -> Result<()> {
+        self.actor
```

Comment:
We can use the `actor::notify` function in the future and remove the `tokio::spaw.`